krux_boto/boto.py

```diff
@@ -198,7 +199,11 @@ def __init__(
             secret_key = DEFAULT['secret_key']()
 
         if region is None:
-            region = DEFAULT['region']()
+            if DEFAULT['region']() is None:
```

Comment:
Nit: You can probably do following to avoid running `DEFAULT['region']()` twice.

```python
if region is None:
    region = DEFAULT['region']()
    if region is None:
        self._logger.warn()
        region = 'us-east-1
```